scripts/legacy_expression_tool/README.md

```diff
@@ -225,8 +234,10 @@ All                                                    1        1   2
 #### Execution with `expressionlanguage` set to `jexlall`
 
 When executing the script with `expressionlanguage` set to `jexlall`, the script will look for all the documents
-containing legacy expressions or the parameter `expressionLanguage`. This would change the number of documents found,
-and the statistics will be different. Running the script with the option set to `jexl` would not have impact on  the statistics. 
+containing legacy expressions (in some of the expression capable fields) or the existence of the `expressionLanguage` field.
+This would change the number of documents found, and the statistics will include extra documents.
+
+Running the script with the option set to `jexl` would not include such extra documents in statistics.
```

Comment:
"statistics will be different" or "impact of statistics" doesn't describe the exact differences or impact.

My suggested wording try to be more precise, although I don't know if I'm fully right.

Please improve if needed.

Reply:
You are right. I agree the rewording describes better that particular scenario.